cli/tsc.rs

```diff
@@ -203,6 +202,16 @@ lazy_static! {
     Regex::new(r#"(\slib\s*=\s*)('|")(.+?)('|")"#).unwrap();
 }
 
+fn warn_ignored_options(
+  maybe_ignored_options: Option<tsc_config::IgnoredCompilerOptions>,
+  config_path: &Path,
+) {
+  if let Some(ignored_options) = maybe_ignored_options {
+    let ignored_opts = ignored_options.to_string();
```

Comment:
Doesn't this happen automatically on formatting?  It should.

Reply:
It does, thanks